packages/cf.js/API_REFERENCE.md

```diff
@@ -1,82 +1,341 @@
-# API V0.0.2
-## `cf.js`
+# CF.js API Specification V0.0.2
+
+### `cfjs` Typescript Package
 
 - `Provider`
     - Properties
         - `nodeProvider: NodeProvider`
+            - Refer to `node-provider` package
     - Instance methods
         - `async getAppInstances(): AppInstance[]`
-        - `createAppFactory(appDefinition: AppDefinition): AppFactory`
```

Comment:
why was `createAppFactory` removed?

Reply:
It wasn't necessary. Now you instantiate an AppFactory with a provider and an AppDefinition, which makes more sense.